libursa/src/kex/secp256k1.rs

```diff
@@ -130,6 +130,7 @@ mod ecdh_secp256k1 {
     use sha2::digest::generic_array::typenum::U32;
     use sha2::Digest;
     use zeroize::Zeroize;
+    #[allow(dead_code)]
```

Comment:
Probably better to remove the dead code. What was coming up?

Reply:
It's not used anywhere, I was just taking a super conservative approach for my changes. I'll go ahead and remove it now.